README.md

```diff
@@ -1,64 +1,147 @@
-# [WIP] Reaction Next Storefront Starterkit
+# Reaction Storefront Next.js Starter Kit
 
-Example storefront application for [Reaction Commerce](https://reactioncommerce.com/).
+_**Note:** This project is a work in progress and should not be used in production at this time._
 
-## Setup
+Reference Storefront application for [Reaction Commerce](https://reactioncommerce.com/).
 
-`git clone https://github.com/reactioncommerce/reaction-next-starterkit.git && cd reaction-next-starterkit`
+## Getting Started
+_Follow steps as necessary. If you already have Reaction installed, you may be able to skip some of these steps._
 
-### Prerequisites
+0. Prerequesites
+- Install [Docker](https://docs.docker.com/install/) and [Docker Compose](https://docs.docker.com/compose/install/). Docker Compose is included in when installing Docker on Mac and Windows, but will need to be installed separately on Linux.
 
-**Creating a local docker network**
+1. Clone the main Reaction repo https://github.com/reactioncommerce/reaction and checkout the `release-1.12.0` branch
+    ```sh
+    git clone git@github.com:reactioncommerce/reaction.git
+    cd reaction
+    git checkout release-1.12.0
 
-First you'll need to create a docker network for the GraphQL service and Storefront to communicate.
+    # change directory to the parent of your reaction install
+    cd ..
+    ```
 
-_**NOTE:** Currently we're using the network name of `reaction-api` but this may change in the future._
+2.
+    ```sh
+    git clone git@github.com:reactioncommerce/reaction-next-starterkit.git
+    ```
 
- 1. Create a network by running `docker network create reaction-api`.
- 2. Run `docker network ls` to verify your new network has been created
+3. Create a local docker network
 
-**Starting Reaction's GraphQL server**
+    You'll need to create a docker network for the GraphQL service and the Reaction Storefront to communicate
+    ```
+    docker network create reaction-api
+    ```
+    You can run `docker network ls` to verify the network has been created.
 
-_**NOTE:** Currently we're using the [release 1.11.0 branch](https://github.com/reactioncommerce/reaction/pull/4151) of Reaction Commerce to create the GraphQl service._
+4. Start Reaction's GraphQL server
+    From your `reaction` directory run
+    ```
+    docker-compose up -d --build
+    ```
+    Optionally, if you don't want to start the Reaction Meteor app, you can just run `docker-compose up -d devserver`
 
- 1. Pull the latest code from Reaction Commerce
- 2. Start the devserver by running `docker-compose up -d devserver` or start both the `devserver` and `reaction` by running `docker-compose up -d`
+5. Generate a Meteor login token
```

Comment:
In step 4, we say it's optional to start up `:3000`, but in step 5, we say you need to get the token from it.

Should we say that you need to start up the full app at least once to get the token? And then subsequent times, you can just start the devserver?